core/src/avm2/globals/flash/display/ScreenMode.as

```diff
@@ -0,0 +1,40 @@
+package flash.display
+{
+  import __ruffle__.stub_getter;
+  import flash.geom.Rectangle;
+
+  // [API("733")] Ruffle doesn't support this API Version
```

Comment:
We have to use _some_ AIR version, otherwise it's available to normal Flash.

Reply:
What version do you suggest?

Reply:
This is from what I helped with the PR, my bad! didn't notice I left the class unversioned
Should probably use AIR 1.0 (`[API(661)]`)